$NetBSD: patch-Makefile,v 1.2 2014/01/22 16:16:21 jperkin Exp $

Don't hardcode PREFIX
Fix the LUA version to match the one installed by pkgsrc
Don't use mkdir and cp directly, use BSD_INSTALL_* abstraction instead

--- lgi/Makefile.orig	2013-03-04 21:10:19.000000000 +0000
+++ lgi/Makefile	2013-07-21 21:56:37.000000000 +0000
@@ -5,8 +5,7 @@
 # License: MIT
 #
 
-PREFIX = /usr/local
-LUA_VERSION=5.1
+LUA_VERSION=@_LUA_DOT_VERSION@
 LUA_LIBDIR = $(PREFIX)/lib/lua/$(LUA_VERSION)
 LUA_SHAREDIR = $(PREFIX)/share/lua/$(LUA_VERSION)
 
@@ -74,11 +73,11 @@
 CORESOURCES = $(wildcard *.lua)
 
 install : $(CORE) $(VERSION_FILE)
-	mkdir -p $(DESTDIR)$(LUA_LIBDIR)/lgi
-	cp $(CORE) $(DESTDIR)$(LUA_LIBDIR)/lgi
-	mkdir -p $(DESTDIR)$(LUA_SHAREDIR)
-	cp ../lgi.lua $(DESTDIR)$(LUA_SHAREDIR)
-	mkdir -p $(DESTDIR)$(LUA_SHAREDIR)/lgi
-	cp $(CORESOURCES) $(VERSION_FILE) $(DESTDIR)$(LUA_SHAREDIR)/lgi
-	mkdir -p $(DESTDIR)$(LUA_SHAREDIR)/lgi/override
-	cp $(OVERRIDES) $(DESTDIR)$(LUA_SHAREDIR)/lgi/override
+	$(BSD_INSTALL_LIB_DIR) $(DESTDIR)$(LUA_LIBDIR)/lgi
+	$(BSD_INSTALL_LIB) $(CORE) $(DESTDIR)$(LUA_LIBDIR)/lgi
+	$(BSD_INSTALL_DATA_DIR) $(DESTDIR)$(LUA_SHAREDIR)
+	$(BSD_INSTALL_DATA) ../lgi.lua $(DESTDIR)$(LUA_SHAREDIR)
+	$(BSD_INSTALL_DATA_DIR) $(DESTDIR)$(LUA_SHAREDIR)/lgi
+	$(BSD_INSTALL_DATA) $(CORESOURCES) $(VERSION_FILE) $(DESTDIR)$(LUA_SHAREDIR)/lgi
+	$(BSD_INSTALL_DATA_DIR) $(DESTDIR)$(LUA_SHAREDIR)/lgi/override
+	$(BSD_INSTALL_DATA) $(OVERRIDES) $(DESTDIR)$(LUA_SHAREDIR)/lgi/override
